app.js

```diff
@@ -1,11 +1,25 @@
 const express = require("express");
 const app = express();
 const getTopicsController = require("./controllers/topics.controller");
+const fs = require("fs/promises");
 
 app.get("/api/topics", getTopicsController);
 
+app.get("/api", (request, response) => {
+  fs.readFile("endpoints.json", "utf-8").then((apiJson) => {
+    response.send(apiJson);
```

Comment:
This logic should be in it's own controller. So we'd just have say app.get("/api", getApi) where 'getApi' (or whatever you want to call it) would live in the controller, sending the endpoints.json file 👍